18/README.md

```diff
@@ -0,0 +1,208 @@
+```
+shortname: [18/TEP]
+name: Transactional Election Process
+type: standard
+status: raw
+editor: Alberto Granzotto <alberto@bigchaindb.com>
+```
+
+
+# Transactional Election Process
+
+## Abstract
+<!-- The abstract is a short (~200 word) description of the technical issue being addressed. -->
+This specification introduces the new concept of **Election**. An Election is an asynchronous process that when successful triggers Network wide changes synchronously (i.e. at the same *block height*). An Election is started by any Validator in the Network, called **Initiatior**. The Election itself and all cast Votes are transactions, hence stored in the blockchain. This enables new Validators to replay Network changes incrementally, while syncing.
+
+## Motivation
+<!--The motivation is critical for BEPs that want to change the BigchainDB protocol. It should clearly explain why the existing protocol BEP is inadequate to address the problem that the BEP solves. BEP submissions without sufficient motivation may be rejected outright.-->
+Changing the shape of a BigchainDB Network at runtime is an important requirement. While this BEP does not address this issue directly, it wants to solve the limitations we had with [Upsert Validators][BEP-3] by giving a tool that can be used this and other situations.
+
+The basic idea is to formalize the concept of an Election storing its data in a [BigchainDB Transaction Spec v2][BEP-13].
+By storing it in a BigchainDB Network, it allows Members to vote asynchronously, and Validators to apply changes synchronously.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations. It MAY describe the impact on data models, API endpoints, security, performance, end users, deployment, documentation, and testing.-->
+
+At any point in time, a Member of a BigchainDB Network can start a new Election.This Member is called **Initiator**.
+
+An Election is a transaction representing the matter of change, and some Vote tokens. The Initiator issues a `CREATE` transaction with the `amount` set to the total number of Validators, and transfer one vote token per Validator using the transaction `outputs`.
+
+At this point the Election starts. Independently, and asynchronously, each Validator can spend its Vote Token to an Election Address to show agreement on the matter of change. The Election Address is the `id` of the first `CREATE` transaction. Once a Vote Token has been transferred to that address, it is not possible to transfer it again, because there private key is not known.
```

Comment:
I don't understand how you can transfer/spend an election token to the 'id' of a CREATE transaction. That will be a hash, but in BigchainDB, one transfers/spends outputs to public keys, not hashes.

Reply:
That's a tricky part I should explain more. Last night I wrote a [note about that](https://github.com/vrde/notes/tree/master/bigchaindb#using-a-public-key-as-a-topic).

A public key can be generated from a private key, or initialized with 32 bytes (256 bits). The `id` of a transaction is a 256 bit long hash. So we can use `id`s as seeds to initialize a public key :champagne:. I need to expand this section to explain this process. Maybe we can have a quick chat with @kansi and you. It can be an `informal` BEP, so it can be reused in other `BEP`s.